### DISCUSSION TITLE
ESP32C6 not working Zigbee communication after esp32 library update 3.0.0-alpha3 to 3.0.3

### DISCUSSION THREAD

**Discussion by MaXeN9818** [espressif/arduino-esp32#10088] [Q&A] [answered]
____________________________________________________________
**IDE: Arduino IDE v.2.3.2**
**OS: Windows 11**
**HW: nanoESP32C6 v1.0 Build 19 Sep 2022** 
**GATEWAY: SONOFF Zigbee 3.0 USB ZBDongle Plus** 
___________________________________________________________

I created a script for ESP32C6 to send data from a pressure switch via Zigbee, and it worked without any problems until now. I wanted to update the script and took the opportunity to update the libraries as well. However, after updating to version 3.0.3 (of course I modified the script to make it compatible), the connection is successful, but after a few seconds I no longer receive data from the ESP. Even if I try to reboot with the reset button, the ESP can no longer connect to my Gateway. Another problem I noticed is that the ESP starts to overheat significantly, which was not the case with the old version.

_I tried the Zigbee Temperature Sensor example, but it gives me the same problem._

_If I go back to the old version it works again._

**Do you have any ideas on what the problem could be? I think the problem is the ESP card too old.**

**Comment by me-no-dev** [comment]
@P-R-O-C-H-Y PTAL

**Comment by P-R-O-C-H-Y** [comment]
Hello @MaXeN9818, I have ordered the SONOFF Zigbee gateway so I will test it. Last time I run the examples ED(esp32c6) and COORDINATOR(esp32h2) all worked fine. As for now we are just providing the Zigbee libraries without any wrapper, can you please open an issue in the [ESP-ZIGBEE-SDK](https://github.com/espressif/esp-zigbee-sdk) repository and ping me also there? Thank you

**Reply by MaXeN9818** [reply]
Hi sorry, I did the test with two ESP32C6 one ED and one ZCZR and it actually worked without problems (a couple of times it failed to connect but nothing serious). So I focused my attention on the Sonoff I flashed the firmware with the same version already installed. Everything seems to work now, without problems. My script and the example one work.

Thanks, and sorry for the inconvenience

**Reply by MaXeN9818** [reply]
No, I have to correct myself again after 2 hours and 30 minutes the system stopped working and I had to restart it and reconnect manually.

On some forums I read that updating the Firmware of the SONOFF gateway can improve stability, I will try
Even if I want to first do a test with the two ESPs, an ED and a ZCZR, but maintaining the connection for several hours

**Reply by P-R-O-C-H-Y** [reply]
Thank you for updates, let us know here any news regarding your test. It's very helpful.
Once my Sonoff gateway and end device arrives, I can also do some testing.

**Comment by MaXeN9818** [comment, marked as answer]
Sorry if this took a while. 
The disconnection problem has not occurred again and now everything seems stable.
I simply flashed the same firmware on the USB dongle and now it seems to work without problems

Thank you
